Classes/ViewHelpers/Uri/AjaxActionViewHelper.php

```diff
@@ -29,6 +29,9 @@
  * index.php?id=123&tx_typoscriptrendering[context]={"record":"tt_content_123","path":"tt_content.list.20.myextension_plugin"}&tx_myextension_plugin[action]=show&tx_myextension_plugin[controller]=Standard&cHash=xyz
  * (depending on the current page and your TS configuration)
  * </output>
+ *
+ * Please note that this only works for plugins that are using the PLUGIN_TYPE_PLUGIN type,
+ * not for PLUGIN_TYPE_CONTENT_ELEMENT plugins.
```

Comment:
Are these constants? Where do they come from and why do you assume this VH does not work for content elements?

What exactly does not work?